server/src/main/resources/rules/rules.js

```diff
@@ -74,7 +76,7 @@ var VCPU_ATTRIBUTE = "vcpu";
 var MULTI_ENTITLEMENT_ATTRIBUTE = "multi-entitlement";
 var STACKING_ID_ATTRIBUTE = "stacking_id";
 var STORAGE_BAND_ATTRIBUTE = "storage_band";
-var ROLE_ATTRIBUTE = "role";
+var ROLES_ATTRIBUTE = "roles";
```

Comment:
Is this an internal change or will this change something about the output of the compliance event that might require katello to change?

Reply:
The output of the syspurpose compliance event won't change, because it is about the compliance of a specific consumer, who can only have a single role specified. This change was required because the Product attribute, as agreed with IT, is called `roles`, and not `role` as seen [here](https://github.com/candlepin/candlepin/blob/7c6859c68821767e45e689eb1278bed090ed85c4/server/src/main/java/org/candlepin/model/Product.java#L135). That is because a Product/SKU can have multiple roles (comma-separated list, similarly to how the `arch` and `addons` attributes work).

@kahowell can you verify that my understanding is correct if possible (this is what I understand from reading the code, and what we've agreed with IT)?